### PR TITLE
Add list of supported operating systems

### PR DESCRIPTION
**Pull request by arthurpetitpierre**
Add list of supported operating systems with links to AMIs

### CODE REVIEW

**Comment by arthurpetitpierre** [comment]
Added Metal support, LSE and removed outdated ubuntu releases

File: os.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Operating Systems available for Graviton based instances
+
+ Name | Version | [LSE Support](https://github.com/aws/aws-graviton-getting-started/blob/master/optimizing.md#locksynchronization-intensive-workload) | Kernel page size | AMI | Metal support | Comment
+------ | ------ | ----- | ----- | ----- | ----- | -----
+Amazon Linux 2 | 2.26-35 | Yes | 4KB | Yes |
```
Comment (AGSaidi) [inline os.md:5]: AMI link missing, yes is wrong column

File: os.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Ubuntu | 20.04 LTS | Yes | 4KB | [focal](https://cloud-images.ubuntu.com/locator/ec2/) | Yes | 
+Ubuntu | 18.04 LTS | [Planned](https://launchpad.net/~rbalint/+archive/ubuntu/arm64-lse) | 4KB | [bionic](https://cloud-images.ubuntu.com/locator/ec2/) | Yes | 
+SuSE | 15 SP2 | Planned | 4KB | [MarketPlace](https://aws.amazon.com/marketplace/pp/B07SPTXBDX) | Yes | 
+Redhat Entreprise Linux | 8.2 | Yes | 64KB | [MarketPlace](https://aws.amazon.com/marketplace/pp/B07T2NH46P) | Yes | 
+~~Redhat Entreprise Linux~~ | ~~7.x~~ | ~~No~~ | ~~64KB~~ | ~~[MarketPlace](https://aws.amazon.com/marketplace/pp/B07KTFV2S8)~~ | | Supported on A1 instances but not on Graviton2 based ones
+CentOS | 8.2.2004 | No | 64KB | [AMIs](https://wiki.centos.org/Cloud/AWS#Images) | Yes | 
+~~CentOS~~ | ~~7.x~~ | ~~No~~ | ~~64KB~~ | ~~[AMIs](https://wiki.centos.org/Cloud/AWS#Images)~~ | | Supported on A1 instances but not on Graviton2 based ones
+Debian | 10 | [Planned](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=956418) | 4KB | [Community](https://wiki.debian.org/Cloud/AmazonEC2Image/Buster) or [MarketPlace](https://aws.amazon.com/marketplace/pp/B085HGTX5J) | Yes |
```
Comment (AGSaidi) [inline os.md:13]: metal support doesn't work yet, there is a patch for the next debian release

File: os.md (hunk 1 above)
Comment (arthurpetitpierre) [reply os.md]: Fixed column + added a page to explain how to use AWS Systems Manager Parameter store to get the latest AMI_IDs.

File: os.md (hunk 2 above)
Comment (arthurpetitpierre) [reply os.md]: Added a link to Debian bug tracker for metal support.

**Pull request merged**